interface/billing/billing_report.php

```diff
@@ -65,59 +52,55 @@
             header("Expires: 0");
             header("Cache-Control: must-revalidate, post-check=0, pre-check=0");
             header("Content-Disposition: attachment; filename=openemr_ofx.ofx");
             header("Content-Type: text/xml");
             echo $ofx->get_OFX();
-            exit;
+            exit();
         }
     }
 }
 
-//global variables:
-$from_date     = isset($_POST['from_date'])  ? $_POST['from_date']  : date('Y-m-d');
-$to_date       = isset($_POST['to_date'  ])  ? $_POST['to_date'  ]  : '';
-$code_type     = isset($_POST['code_type'])  ? $_POST['code_type']  : 'all';
-$unbilled      = isset($_POST['unbilled' ])  ? $_POST['unbilled' ]  : 'on';
+// global variables:
+$from_date = isset($_POST['from_date']) ? $_POST['from_date'] : date('Y-m-d');
+$to_date = isset($_POST['to_date']) ? $_POST['to_date'] : '';
+$code_type = isset($_POST['code_type']) ? $_POST['code_type'] : 'all';
+$unbilled = isset($_POST['unbilled']) ? $_POST['unbilled'] : 'on';
 $my_authorized = isset($_POST["authorized"]) ? $_POST["authorized"] : '';
 
-
 // This tells us if only encounters that appear to be missing a "25" modifier
 // are to be reported.
-$missing_mods_only = (isset($_POST['missing_mods_only']) && !empty($_POST['missing_mods_only']));
+$missing_mods_only = (isset($_POST['missing_mods_only']) && ! empty($_POST['missing_mods_only']));
 
 $left_margin = isset($_POST["left_margin"]) ? $_POST["left_margin"] : $GLOBALS['cms_left_margin_default'];
-$top_margin  = isset($_POST["top_margin"]) ? $_POST["top_margin" ] : $GLOBALS['cms_top_margin_default'];
-
-$ofrom_date  = $from_date;
-$oto_date    = $to_date;
-$ocode_type  = $code_type;
-$ounbilled   = $unbilled;
+$top_margin = isset($_POST["top_margin"]) ? $_POST["top_margin"] : $GLOBALS['cms_top_margin_default'];
+if ($ub04_support) {
+    $left_ubmargin = isset($_POST["left_ubmargin"]) ? $_POST["left_ubmargin"] : $GLOBALS['left_ubmargin_default'];
+    $top_ubmargin = isset($_POST["top_ubmargin"]) ? $_POST["top_ubmargin"] : $GLOBALS['top_ubmargin_default'];
+}
+$ofrom_date = $from_date;
+$oto_date = $to_date;
+$ocode_type = $code_type;
+$ounbilled = $unbilled;
 $oauthorized = $my_authorized;
 ?>
-
+<!DOCTYPE html >
 <html>
 <head>
-<?php if (function_exists('html_header_show')) {
-    html_header_show();
-} ?>
-<link rel="stylesheet" href="<?php echo $css_header; ?>" type="text/css">
-<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-datetimepicker-2-5-4/build/jquery.datetimepicker.min.css">
 
-<style>
-.subbtn { margin-top:3px; margin-bottom:3px; margin-left:2px; margin-right:2px }
-</style>
-<script>
+<?php Header::setupHeader(['datetime-picker','common']); ?>
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap-3-3-4/dist/css/bootstrap.min.css">
+<script src="<?php echo $GLOBALS['assets_static_relative']; ?>/bootstrap-3-3-4/dist/js/bootstrap.min.js" type="text/javascript"></script>
```

Comment:
rec trying to remove above 2 lines again. i confirmed that these two lines are already included in the Header::setupHeader call above (in browser on the page, did a Frame->View Source).